.github/workflows/ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
```

Comment:
we don't have a main branch, it doesn't hurt to have this just in case we decide to rename it

Reply:
Yeah, that was my (lazy) thinking 😂